### PR TITLE
[full-ci] Add remove/add users to group batch actions 

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
Adds new batch actions for adding and removing group assignments to the users list in the admin settings.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/8559
-  Fixes https://github.com/owncloud/web/issues/8558
- Fixes https://github.com/owncloud/web/issues/8472

## Screenshots:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/33450/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33450/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2023-3-8-11-46-11.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33450/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2023-3-8-11-46-43.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33450/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-carol-2023-3-8-11-46-48.zip 


</p></details>
:boom: The e2e-oc10 tests pipeline failed. The build has been cancelled.

File: packages/web-app-admin-settings/src/views/Users.vue
Comment (kulmann) [inline packages/web-app-admin-settings/src/views/Users.vue:108]: Needs pluralization. String doesn't work for single selected user.

File: packages/web-app-admin-settings/src/views/Users.vue
Comment (kulmann) [inline packages/web-app-admin-settings/src/views/Users.vue:119]: Same here, needs pluralization. String doesn't work for single selected user.

File: packages/web-app-admin-settings/src/components/Users/GroupsModal.vue
Comment (kulmann) [inline packages/web-app-admin-settings/src/components/Users/GroupsModal.vue:4]: The message seems to be unused when you overwrite the content slot. 🙈 💩  i.e. the message doesn't appear at all in the modal.

File: packages/web-app-admin-settings/src/components/Users/GroupsModal.vue
Comment (kulmann) [inline packages/web-app-admin-settings/src/components/Users/GroupsModal.vue:39]: Please sort the groups in the same manner as in the right sidebar user edit form.

File: packages/web-app-admin-settings/src/components/Users/GroupsModal.vue
Comment (kulmann) [inline packages/web-app-admin-settings/src/components/Users/GroupsModal.vue:12]: Possible to reuse the `GroupSelect.vue` component instead? Right now the two group assignments (batch or single user in sidebar) look very different. Would prefer to have them look the same.

**Review by kulmann — APPROVED** [review]
Feature works as expected. Thanks for the detour with batch actions in the AppTemplate and for already using the composable syntax for the actions! :heart:

Added some comments about things that need to be fixed before merge. Please don't merge before those are done as well. :-) Approving because I'm OOTO tomorrow and on friday and don't want to block this (and it's only complaints about visual defects)

**Comment by kulmann** [comment]
Converted to draft so that this doesn't get merged accidentally.

**Comment by JanAckermann** [comment]
THX!

**Comment by JanAckermann** [comment]
Added a ocis ticket to minimize request https://github.com/owncloud/ocis/issues/5783

**Pull request merged**
